[FEATURE] Update clés de trad pour réconciliation SCO FWB (PIX-13943)

pix-service-prod opened the pull request:
Exported the most recent locale files from Phrase

pix-service-prod committed:
Updated translations from Phrase

phrase: update mon-pix/translations/nl.json to latest state
phrase: update mon-pix/translations/es.json to latest state

pix-bot-github commented:
Une fois les applications déployées, elles seront accessibles via les liens suivants :
  * [App (.fr)](https://app-pr9897.review.pix.fr)
  * [App (.org)](https://app-pr9897.review.pix.org)
  * [Orga (.fr)](https://orga-pr9897.review.pix.fr)
  * [Orga (.org)](https://orga-pr9897.review.pix.org)
  * [Certif (.fr)](https://certif-pr9897.review.pix.fr)
  * [Certif (.org)](https://certif-pr9897.review.pix.org)
  * [Junior](https://junior-pr9897.review.pix.fr)
  * [Admin](https://admin-pr9897.review.pix.fr)
  * [API](https://api-pr9897.review.pix.fr/api/)
  * [Audit Logger](https://pix-audit-logger-review-pr9897.osc-fr1.scalingo.io/api/)

Les variables d'environnement seront accessibles via les liens suivants :
  * [scalingo front](https://dashboard.scalingo.com/apps/osc-fr1/pix-front-review-pr9897/environment)
  * [scalingo api](https://dashboard.scalingo.com/apps/osc-fr1/pix-api-review-pr9897/environment)
  * [scalingo audit-logger](https://dashboard.scalingo.com/apps/osc-fr1/pix-audit-logger-review-pr9897/environment)

Faraopix commented:
@Jeyffrey  J'ai l'impression qu'il y a des modifs au niveau des clés concernant la page de fin de parcours 👀

AndreiaPena commented on mon-pix/translations/es.json:
```diff
@@ -516,12 +514,13 @@
           "checkbox-label": "Al marcar esta casilla, reconozco que he leído estas normas y me comprometo a cumplirlas.",
           "list": {
-            "miscellaneous": "'<strong>'Movilizar cualquier otro medio que proporcione directamente la respuesta'</strong>', sin haber realizado el trabajo requerido por la instrucción.",
+            "other": "'<strong>'Movilizar cualquier otro medio que proporcione directamente la respuesta'</strong>', sin haber realizado el trabajo requerido por la instrucción.",
```
ça ne devrait pas repasser en `other` 😅 mais rester en `miscellaneous`
On va regarder coté certif ce qui coince sur Phrase

AndreiaPena replied:
Ok ça devrait être bon :) est-ce qu'il est possible de repush @Faraopix ?

pix-service-prod committed:
Updated translations from Phrase

phrase: update mon-pix/translations/nl.json to latest state
phrase: update mon-pix/translations/es.json to latest state

AndreiaPena reviewed (approved):
top ok pour certif !

Jeyffrey commented on mon-pix/translations/es.json:
```diff
@@ -1696,24 +1681,6 @@
         "starsAcquired": "{acquired, plural, =0 {ninguna estrella obtenida} =1 {# estrella obtenida} other {# estrellas obtenidas}} de {total}",
         "title": "Detalles de las competencias"
       },
-      "tabs": {
```
C'est un développement en cours côté eval https://github.com/1024pix/pix/pull/9771/files#diff-043535b8fcbd2cca86200d12575216ca38c089d260a044189fb692f5f15c859c

On a besoin de cette clé et de ses enfants :)

Faraopix replied:
@Jeyffrey  je ne comprends pas pourquoi la sync phrase/github les fait sauter du coup 🤔

Jeyffrey replied:
Je ne sais pas du tout 🤷‍♂️

Jeyffrey commented on mon-pix/translations/es.json:
```diff
@@ -1634,10 +1622,7 @@
       "already-shared": "¡Gracias, tus resultados se han enviado correctamente!",
       "badge-card": {
         "acquired": "Aprobado",
-        "acquired-full": "Resultado temático aprobado",
-        "certifiable": "Certificado",
         "not-acquired": "No has aprobado",
-        "not-acquired-full": "Resultado temático no has aprobado",
```
Pareil ici, nous avons besoin de ces clés de trad
https://github.com/1024pix/pix/pull/9780/files#diff-043535b8fcbd2cca86200d12575216ca38c089d260a044189fb692f5f15c859c

Jeyffrey commented on mon-pix/translations/es.json:
```diff
@@ -42,8 +42,6 @@
       "lets-go": "¡Empecemos!",
       "quit": "Salir",
-      "show-less": "Mostrar menos",
-      "show-more": "Mostrar más",
```
Pareil ici https://github.com/1024pix/pix/pull/9780/files#diff-043535b8fcbd2cca86200d12575216ca38c089d260a044189fb692f5f15c859c

pix-service-prod committed:
Updated translations from Phrase

phrase: update mon-pix/translations/nl.json to latest state
phrase: update mon-pix/translations/es.json to latest state
